src/verifier.js

```diff
@@ -0,0 +1,82 @@
+const fetch = require('node-fetch')
+const hash = require("hash.js")
+
+// hash the parameter values to be sent to the verifier
+function hashParams(params) {
+    let hashedParams = {}
+    Object.keys(params).map(key => {
+        hashedParams[key] = hash.sha256().update(params[key]).digest('hex')
+    })
+    return hashedParams
+}
+
+//Call the Verifier to verify the client request and return an ore-access-token to access a particular right
+async function getAccessTokenFromVerifier(verifierEndpoint, instrument, right, hashedParams) {
+    let errorMessage;
+    let result;
+    let signature;
+
+    try {
+        signature = await this.signVoucher(instrument.id);
+    } catch (error) {
+        throw new Error(`${error.message}`);
```

Comment:
This isn't doing anything extra. You can just remove the entire try/catch block, and accomplish the same thing.